pkg/cloudcore/apis/config/default.go

```diff
@@ -0,0 +1,75 @@
+package config
```

Comment:
can we add LICENSE HEAD here ?

@fisherxu @kevin-wangzefeng

Reply:
We need to use a licensing tool such as "fossa" to ensure we follow all license guidelines

Reply:
Applicable to all new places

Reply:
ok